Documentation/ApiOverview/ErrorAndExceptionHandling/Configuration/Index.rst

```diff
@@ -10,232 +10,64 @@
 Configuration
 =============
 
-All configuration options related to error and exception handling are
-part of :php:`$GLOBALS['TYPO3_CONF_VARS'][SYS]`:
+You can configure the most important settings for production or debug error
+handling in the presets:
 
-.. t3-field-list-table::
- :header-rows: 1
+:guilabel:`Admin Tools > Settings > Configuration Presets > Debug Settings`
 
- - :Key,20: Key
-   :Data type,20: Data type
-   :Description,60: Description
+For more fine-grained error handling you can change diverse settings in:
 
- - :Key:
-         displayErrors
-   :Data type:
-         integer
-   :Description:
-         Configures whether PHP errors should be displayed.
+:guilabel:`Admin Tools > Settings > Configure Installation-Wide Options > SYS`
 
-         - 0 = Do not display any PHP error messages. Overrides the value of
-           "exceptionalErrors" and sets it to 0 (= no errors are turned into
-           exceptions),the configured :php:`productionExceptionHandler` is used as
-           exception handler
+It is also possible to write the changes into the configuration file
+:file:`typo3conf/LocalConfiguration.php` or
```

Comment:
```suggestion
:file:`config/system/settings.php` or
```

As we merge into main, this should reflect v12. In the backport it can then be adjusted to LocalConfiguration.php